x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/action/TransportDeleteExpiredDataActionTests.java

```diff
@@ -50,11 +50,11 @@
     private static class DummyDataRemover implements MlDataRemover {
 
         public void remove(
-            float requestsPerSec,
-            ActionListener<Boolean> listener,
-            Supplier<Boolean> isTimedOutSupplier
+                float requestsPerSec,
+                ActionListener<Boolean> listener,
+                BooleanSupplier isTimedOutSupplier
```

Comment:
I think this indentation change does not match our conventions, can you remove the extra indentation?